assets/src/edit-story/app/media/media3p/api/apiFetcher.js

```diff
@@ -162,14 +170,41 @@ class ApiFetcher {
     });
   }
 
-  async fetch({ params, path }) {
+  /**
+   * Perform a POST request to the Media3P API to register a usage.
+   * If the parameters are invalid or the server does not return a 200 response,
+   * an error is thrown.
+   *
+   * @param {Object} obj - An object with the options for the request.
+   * @param {string} obj.payload Payload to be posted to register media usage.
+   * @return {Promise<Object>} The response from the API.
+   */
+  async registerUsage({ payload }) {
+    validateRegisterUsagePayload(payload);
+
+    const params = [
+      ['payload', payload],
+      ['key', API_KEY],
+    ].filter((entry) => Boolean(entry[1]));
+
+    // eslint-disable-next-line no-return-await
```

Comment:
I think this lint can be fixed by removing both 'async' (from the function definition) and 'await' keywords?

Reply:
This lead to a change as the method was crashing due to wrong validation instead of a failure in the promise (as I also wanted to change all to stay consistent). So I kept as is. Can be investigated separately.

Reply:
That makes sense.